tests/queries/0_stateless/02346_gin_index_match_predicate.sql

```diff
@@ -12,7 +12,7 @@ CREATE TABLE tab
 )
 ENGINE = MergeTree
 ORDER BY id
-SETTINGS index_granularity = 1;
+SETTINGS index_granularity = 1, min_bytes_for_full_part_storage = 0;
```

Comment:
To make this less confusing for the next guy who works on this, please add a comment:

```sql
SETTINGS index_granularity = 1,
         min_bytes_for_full_part_storage = 0; -- GIN indexes currently don't work with packed parts
```

(same in other files)